elasticsearch_exporter.go

```diff
@@ -429,3 +435,47 @@ func main() {
 	})
 	log.Fatal(http.ListenAndServe(*listenAddress, nil))
 }
+
+func createElasticSearchTlsConfig(pemFile, pemCertFile, pemPrivateKeyFile string) (*tls.Config) {
+	if (len(pemFile) > 0) {
```

Comment:
This one can be inverted to `if (len(pemFile) <= ) { return nil }` so that we can get rid of the extra else at the bottom of this `func`. I think that applies for a few other conditions in this func too. It might be nitpicking, but definitely something I'd do different.